module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -4,101 +4,102 @@
 ##### Each table represents a different library/tool and the corresponding commands. 
 > Please note that these tables are not intended to tell you all the information you need to know about each command. 
 > 
-> The hyperlinks found in each piece of code will take you to the documentation for further information on the usage of each command. 
+> The hyperlinks found in each piece of code will take you to the documentation for further information on the usage of each command. Please be aware that the documentation will generally provide information about the given function's most current version, which will usually (but not always!) match what you have installed on your machine. If you have certain older versions or R or other R packages, the documentation may differ from what you have installed.
```

Comment:
To ease future revisions, we try to follow a one sentence per line convention (you may see this abbreviated as OSPL). I have changed that here, as well as making a small change since I know that rdrr does not always have the latest version. 
```suggestion
> The hyperlinks found in each piece of code will take you to the documentation for further information on the usage of each command.
Please be aware that the documentation will generally provide information about the given function's most current version (or a recent version, depending on how often the documentation site is updated).
This will usually (but not always!) match what you have installed on your machine.
If you have a different version of R or other R packages, the documentation may differ from what you have installed.
```